usr/share/rear/backup/BORG/default/500_make_backup.sh

```diff
@@ -12,6 +12,36 @@ if [ ! -r "$TMP_DIR/backup-include.txt" ]; then
     Error "Can't find include list"
 fi
 
+function check_BORG_create_return_code() {
+
+    borg_create_exit_code=$1
+
+    # 'borg_create' function returns one of the following codes:
+    #   0     success
+    #   1     warning (operation reached its normal end, but there were warnings)
+    #   2     error (like a fatal error)
+    #   128+N killed by signal
+    # They corresponde to the borg return codes:
+    # https://borgbackup.readthedocs.io/en/stable/usage/general.html#return-codes
+    #
+    # Warning (rc=1) can happen if a file changed while backing it up.
+    # $BORGBACKUP_IGNORE_WARNING="yes" makes rear ignoring warnings (rc=1).
+    if test "$BORGBACKUP_IGNORE_WARNING" = "yes" && test $borg_create_exit_code -eq 1; then
+        LogUserOutput "[BORG rc=1] borg create backup operation completed with at least one warning message that was ignored (see rear log file)"
```

Comment:
In ReaR we have the 'is_true' function,
see the comments usr/share/rear/lib/global-functions.sh
how to use it which starts currently online at
https://github.com/rear/rear/blob/master/usr/share/rear/lib/global-functions.sh#L95

So I would suggest to use
```
if is_true "$BORGBACKUP_IGNORE_WARNING" && test $borg_create_exit_code -eq 1 ; then
```